content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -210,6 +210,26 @@ kubectl apply -f my-private-reg-pod.yaml
 kubectl get pod private-reg
 ```
 
+{{< note >}}
+In case the the Pod fails to start with the status `ImagePullBackOff`, view the Pod events:
+```shell
+kubectl describe pod private-reg
+```
+
+If the `FailedToRetrieveImagePullSecret` is present, Kubernetes can't find the `imagePullSecret` with name `<secret name>` specified in the pod manifest. Make sure that it exists and it's spelled properly.
```

Comment:
```suggestion
If you then see an event with the reason set to `FailedToRetrieveImagePullSecret`,
Kubernetes can't find a Secret with name (`regcred`, in this example).
If you specify that a Pod needs image pull credentials, the kubelet checks that it can
access that Secret before attempting to pull the image.

Make sure that the Secret you have specified exists, and that its name is spelled properly.
```